lib/elixir/lib/module.ex

```diff
@@ -890,42 +905,58 @@ defmodule Module do
 
     check_impls_for_overridable(module, tuples)
 
-    :lists.foreach(fn
-      {function_name, arity} = tuple when is_atom(function_name) and is_integer(arity) and arity >= 0 and arity <= 255 ->
-        case :elixir_def.take_definition(module, tuple) do
-          false ->
-            raise ArgumentError,
-                  "cannot make function #{function_name}/#{arity} overridable because it was not defined"
-          clause ->
-            neighbours =
-              if :elixir_config.get(:bootstrap) do
-                []
-              else
-                Module.LocalsTracker.yank(module, tuple)
-              end
-
-            overridable_definitions = :elixir_overridable.overridable(module)
-            count = case :maps.find(tuple, overridable_definitions) do
-              {:ok, {count, _, _, _}} -> count + 1
-              :error -> 1
-            end
-            overridable_definitions = :maps.put(tuple, {count, clause, neighbours, false}, overridable_definitions)
-            :elixir_overridable.overridable(module, overridable_definitions)
-        end
+    :lists.foreach(
+      fn
```

Comment:
Please extract the function  to a variable in the previous line and reformat the code.

Reply:
How exactly should I go about this? Should I 

`def func do 
{...}
end
`
within the make_overridable function scope, or outside its scope?

Or simply move the function a line higher, give it a name, and then stick it into :lists.foreach/2?

Reply:
> Or simply move the function a line higher, give it a name, and then stick it into :lists.foreach/2?

This. Just assign it to a variable right before.